[2023-11-24T00:04:54.650+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-23T00:00:00+00:00 [queued]>
[2023-11-24T00:04:54.658+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-23T00:00:00+00:00 [queued]>
[2023-11-24T00:04:54.658+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-24T00:04:54.664+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): extract> on 2023-11-23 00:00:00+00:00
[2023-11-24T00:04:54.672+0000] {standard_task_runner.py:57} INFO - Started process 9656 to run task
[2023-11-24T00:04:54.676+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-23T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpkt6wx5z6']
[2023-11-24T00:04:54.682+0000] {standard_task_runner.py:85} INFO - Job 76: Subtask extract
[2023-11-24T00:04:54.734+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-23T00:00:00+00:00 [running]> on host 1df7871f6344
[2023-11-24T00:04:55.549+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-23T00:00:00+00:00'
[2023-11-24T00:04:55.550+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-24T00:04:55.551+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl --create-dirs -o /opt/***/data/2023-11-23/downloads/outcomes_2023-11-23.csv https://data.austintexas.gov/api/views/9t4d-g238/rows.csv']
[2023-11-24T00:04:55.569+0000] {subprocess.py:86} INFO - Output:
[2023-11-24T00:04:55.598+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-11-24T00:04:55.599+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-11-24T00:05:11.379+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:09 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:10 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:11 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:12 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:13 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:14 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:15 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:15 --:--:--     0
[2023-11-24T00:05:11.387+0000] {subprocess.py:93} INFO - curl: (52) Empty reply from server
[2023-11-24T00:05:11.395+0000] {subprocess.py:97} INFO - Command exited with return code 52
[2023-11-24T00:05:11.439+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 52.
[2023-11-24T00:05:11.446+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=extract, execution_date=20231123T000000, start_date=20231124T000454, end_date=20231124T000511
[2023-11-24T00:05:11.463+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 76 for task extract (Bash command failed. The command returned a non-zero exit code 52.; 9656)
[2023-11-24T00:05:11.521+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-24T00:05:11.558+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
